serial_no: XUCAS1716012 #XDCAB1944004 #XUCAS1716015 #XUCAS1716011  
cam_name: "camera"
auto_exposure: true
exposure_time: 50000
gain: 0
auto_exposure_limit: 25000
auto_exposure_priority: 1.0
auto_exposure_level_pct: 30
auto_wb: true
yaml_url: "package://ximea_camera/calibration/intrinsic.yaml"
image_data_format: "XI_MONO8" # "XI_RAW8"  #XI_RGB32, XI_MONO8

##
# mx200
# downsample_factor 4, no roi : ~12fps on mx200 camera
# downsample_factor: 4 # default is 0 or 1 -> no downsampling
# roi_width: 1280
# roi_height: 960
#
# downsample_factor 4, with roi : ~18fps on mx200 camera
# roi_left: 136     # ((5120 / 4) - 1008) / 2
# roi_top: 80       # ((3840 / 4) - 800) / 2
# roi_width: 1008   # shall be mod 16 ?
# roi_height: 800   # use mod 32 ?
#
# downsample_factor 4, with roi : ~19fps on mx200 camera
# roi_left: 0       # ((5120 / 4) - 1280) / 2
# roi_top: 176      # ((3840 / 4) - 608) / 2
# roi_width: 1280   # shall be mod 16 ?
# roi_height: 608   # use mod 32 ?
#
# no downsampling: ~1fps on mx200 camera
# downsample_factor: 1 # default is 0 or 1 -> no downsampling
# roi_width: 5120
# roi_height: 3840

##
# mx124 - xiApi only seems to support downscale upto 2
downsample_factor: 2
# so for mx124 we will have to rely on roi to create smaller images (or do conversion to mono ourselves)
# this yields at least 20fps
roi_left: 512
roi_top: 448
roi_width: 1024
roi_height: 608
